frontend/src/pages/tracker-report.page.tsx

```diff
@@ -0,0 +1,253 @@
+import { useLocalization } from "@fluent/react";
+import { NextPage } from "next";
+import { useEffect, useState } from "react";
+import styles from "./tracker-report.module.scss";
+import logoType from "../../../static/images/fx-private-relay-logotype-dark.svg";
+import logo from "../../../static/images/placeholder-logo.svg";
+import { PageMetadata } from "../components/layout/PageMetadata";
+import {
+  HideIcon,
+  InfoFilledIcon,
+  InfoTriangleIcon,
+} from "../components/Icons";
+import Link from "next/link";
+import { FaqAccordion } from "../components/landing/FaqAccordion";
+
+// Paste this in your browser console to get a report URL:
+// { let url = new URL("http://localhost:3000/tracker-report"); url.hash = JSON.stringify({ sender: "email@example.com", received_at: Date.now(), trackers: { "ads.facebook.com": 1, "ads.googletagmanager.com": 2 } }); url.href }
+// This generates the following URL:
+// http://localhost:3000/tracker-report#{%22sender%22:%22email@example.com%22,%22received_at%22:1655288077484,%22trackers%22:{%22ads.facebook.com%22:1,%22ads.googletagmanager.com%22:2}}
+
+type ReportData = {
+  sender: string;
+  received_at: number;
+  trackers: Record<string, number>;
+};
+
+const TrackerReport: NextPage = () => {
+  const { l10n } = useLocalization();
+  const [reportData, setReportData] = useState<ReportData | null>();
+
+  useEffect(() => {
+    function updateReportData() {
+      setReportData(parseHash(window.location.hash));
+    }
+    updateReportData();
+    window.addEventListener("hashchange", updateReportData);
+    return () => {
+      window.removeEventListener("hashchange", updateReportData);
+    };
+  }, []);
+
+  if (typeof reportData === "undefined") {
+    return (
+      <div className={styles.loading}>
+        {l10n.getString("trackerreport-loading")}
+      </div>
+    );
+  }
+  if (reportData === null) {
+    return (
+      <div className={styles["load-error"]}>
+        {l10n.getString("trackerreport-load-error")}
+      </div>
+    );
+  }
```

Comment:
Is there a way to preview the loading / error states?

Reply:
Oh good point, added that to the original description:

> To preview the error state, [mangle or remove the hash parameter](https://deploy-preview-2104--fx-relay-demo.netlify.app/tracker-report/#invalid-hash).
> 
> For the loading state, use a very slow computer :P Alternatively, add a breakpoint to the `containsReportData` function in `tracker-report.page.tsx`.

![image](https://user-images.githubusercontent.com/4251/174753819-34604faa-6ff7-4d7d-8420-36f7d3d8aec4.png)

Reply:
@codemist If you're asking about network throttling, you can also mimic a slow connection on Chrome  and Firefox using the network panel tools: 

**Chrome:**
<img width="716" alt="image" src="https://user-images.githubusercontent.com/3924990/174882897-14e45fd8-2eeb-4ba1-ac58-f79854f1cc86.png">

**Firefox**
<img width="498" alt="image" src="https://user-images.githubusercontent.com/3924990/174883035-767ab228-dd72-4181-acbd-d81fa19c7da2.png">